apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/GrunnlagService.kt

```diff
@@ -428,3 +509,45 @@ class LaastGrunnlagKanIkkeEndres(val behandlingId: UUID) :
             låst til en versjon av grunnlag (behandlingId=$behandlingId)
             """,
     )
+
+data class PersonopplysningerResponse(
+    val innsender: Personopplysning,
+    val soeker: Personopplysning,
+    val avdoede: List<Personopplysning>,
+    val gjenlevende: List<Personopplysning>,
+)
+
+data class Personopplysning(
+    val opplysningType: Opplysningstype,
+    val id: UUID,
+    val kilde: GenerellKilde,
+    val opplysning: PersonInformasjon,
+)
+
+data class GenerellKilde(
+    val type: String,
+    val tidspunkt: Tidspunkt,
+    val detalj: String? = null,
+)
+
+data class PersonInformasjon(
```

Comment:
Bakgrunnen for denne, som er 99% kopi av `PdlPerson`, var å gjøre noen flere felter nullable - i kontekst av ukjente personer. Men mulig den type info heller bør være egne felt, så vi slipper denne dupliseringen.